metricbeat/module/kafka/consumergroup/config.go

```diff
@@ -0,0 +1,37 @@
+package consumergroup
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/elastic/beats/libbeat/outputs"
+)
+
+type metricsetConfig struct {
+	Retries  int                `config:"retries" validate:"min=0"`
+	Backoff  time.Duration      `config:"backoff" validate:"min=0"`
+	TLS      *outputs.TLSConfig `config:"ssl"`
+	Username string             `config:"username"`
+	Password string             `config:"password"`
+	ClientID string             `config:"client_id"`
+
+	Groups []string `config:"groups"`
+	Topics []string `config:"topics"`
+}
+
+var defaultConfig = metricsetConfig{
```

Comment:
We should move this to the kafka module later as I think it is the same for both metricsets.